### DISCUSSION TITLE
Organization / Repository Shortcuts

### DISCUSSION THREAD

**Discussion by girish-io** [community/community#14804] [Other Feature Feedback, Questions, & Ideas]
Hey everyone,

I'm in a couple of organizations and have lots of repositories to manage (some more often than others). Because of this, it's quite a hassle to find the right ones without switching through different organizations and browsing the repositories.

It would be awesome to see a feature where we can pin / create shortcuts for organizations and repositories on the main Github page.

**Comment by ghost** [comment]
@girish-io Hi! I thought the same as you thought, here [Add bookmark in organizations #9452](https://github.com/github/feedback/discussions/9452) -  So... the difference is that I would like a search filter on all the organizations I follow. Although the requests are different, the purpose would be the same Since after pinning organizations like pins, you may want to look for an organization that hasn't been pinned

**Reply by girish-io** [reply]
That sounds pretty useful as well! I hope it gets implemented one day 😃
